quirks/password-rules.json

```diff
@@ -198,6 +198,9 @@
     "ssa.gov": {
         "password-rules": "required: lower; required: upper; required: digit; required: [~!@#$%^&*];"
     },
+    "store.steampowered.com": {
+        "password-rules": " minlength: 6; required: lower; required: upper; required: digit;"
```

Comment:
Can you remove the leading space?

And no other characters are allowed?

Reply:
Special characters (~!@#$%^&*) are allowed but not required

Reply:
Then this might be a better rule:

```
minlength: 6; required: lower; required: upper; required: digit; allowed: [~!@#$%^&*];
```